test.js

```diff
@@ -8,6 +8,9 @@ tester.run('apostrophe', rule, {
 		'I’m looking forward',
 		'rock’n’roll',
 		'my sisters’ friends’ investments',
+		'Summer of ’69',
+		'Personal best: 00\'03"48',
+		'Latitude: 49° 53\' 08"',
```

Comment:
It doesn't break exactly, but you're right in that it would incorrectly suggest `’` (`U+2019`) instead of the correct `‘` (`U+2018`) for the `'33` segment.

Then again, the plugin doesn't have any specific pre-existing support for single quotation mark pairs, so suggesting `‘` (`U+2018`) would never occur anyway, regardless of the film title.

While this PR would introduce a technically incorrect suggestion, the current version would correct the `Postcards'` into `Postcards’` (`U+2019`), and leave the `'33` alone with a straight apostrophe `'`, which feels more undesirable to me.

Let me know if you have any suggestions!